include/circt/Dialect/Calyx/CalyxLoweringUtils.h

```diff
@@ -486,6 +519,9 @@ class ComponentLoweringStateInterface {
 
   /// A mapping between the callee and the instance.
   llvm::StringMap<calyx::InstanceOp> instanceMap;
+
+  /// A json file to store external global memory data
```

Comment:
Please provide some context as to why a JSON file is needed from someone not as familiar with the native Calyx compiler; perhaps there is some documentation you can link to.